src/tests/nativeaot/SmokeTests/PInvoke/PInvokeNative.cpp

```diff
@@ -649,6 +649,31 @@ DLL_EXPORT void* __stdcall GetFunctionPointer()
     return (void*)&SetLastErrorFunc;
 }
 
+#ifdef TARGET_WINDOWS
+class IComInterface: public IUnknown
+{
+public:
+    virtual HRESULT DoWork(int param) = 0;
```

Comment:
```suggestion
    virtual HRESULT STDMETHODCALLTYPE DoWork(int param) = 0;
```
to follow COM rules. (It is required for x86.)